digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/TrafficSignCsvImporter.scala

```diff
@@ -139,6 +136,12 @@ class TrafficSignCsvImporter(roadLinkServiceImpl: RoadLinkService, eventBusImpl:
   private val codeValueFieldMappings = Map(
     "liikennemerkin tyyppi" -> "trafficSignType"
   )
+
+  val speedLimit = Seq(20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120)
+  val speedLimitSigns = Seq(SpeedLimitSign, EndSpeedLimit, SpeedLimitZone, EndSpeedLimitZone)
```

Comment:
For the speedLimitSigns don't we have the def isSpeedLimit on them to check if they are a speedlimit?